testbed/tests/e2e_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tests contains test cases. To run the tests go to tests directory and run:
+// TESTBED_CONFIG=local.yaml go test -v
+
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"strconv"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/open-telemetry/opentelemetry-service/testbed/testbed"
+)
+
+func TestBallastMemory(t *testing.T) {
+	tests := []struct{
+		ballastSize uint32
+		maxRSS uint32
+	}{
+		{100, 50},
+		{500, 70},
+		{1000, 100},
+	}
+
+	for _, test := range tests {
+		tc := testbed.NewTestCase(t, testbed.WithSkipResults())
```

Comment:
Why is this test skipped?

Reply:
This does not test anything related to performance and the results generated are meaningless. The test is actually only using the testbed because it has already implemented start/stopping the agent as a process. Other than that, it doesn't really need anything else from the testbed.

Reply:
Basically it produces this:
```
# Test Results
Started: Tue, 02 Jul 2019 05:44:28 +0530

Test                                    |Result|Duration|CPU Avg%|CPU Max%|RAM Avg MiB|RAM Max MiB|Sent Spans|Received Spans
----------------------------------------|------|-------:|-------:|-------:|----------:|----------:|---------:|-------------:
TestBallastMemory                       |PASS  |      0s|     0.0|     0.0|          0|          0|         0|             0
TestBallastMemory                       |PASS  |      0s|     0.0|     0.0|          0|          0|         0|             0
TestBallastMemory                       |PASS  |      0s|     0.0|     0.0|          0|          0|         0|             0
```

Reply:
OK, makes sense.